proof/invariant-abstract/ARM/ArchVSpaceEntries_AI.thy

```diff
@@ -436,7 +436,7 @@ lemma as_user_valid_pdpt_objs[wp]:
 
 crunch valid_pdpt_objs[wp]: send_signal, send_ipc "valid_pdpt_objs"
   (wp: get_sched_context_wp mapM_wp' maybeM_inv hoare_vcg_if_lift2 hoare_drop_imps
-       transfer_caps_loop_pres
+       transfer_caps_loop_pres whileLoop_wp'
```

Comment:
I think when we were discussing adding this rule to the `wp` set or `crunch_wps`, the consensus was that because these rules are currently used in so few places and there are enough instances where `valid_whileLoop` is actually necessary, it'd be too confusing to add `whileLoop_wp'` to the `wp` set

Reply:
Yeah, that's my recollection. It's not a safe wp rule so if it was in `wp` then it would be very confusing when things go wrong. In practice I think it would actually be ok to have in `crunch_wps`, but in my mind loops are generally special enough that it's ok for them to be a bit more manual.

Reply:
It's already in `crunch_wps`!